src/components/Table.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import './Table.css';
+import { Icon } from '@iconify/react';
+import externalLinkOutline from '@iconify-icons/eva/external-link-outline';
+
```

Comment:
I can't remember if we decided whether or not this component would start getting the data from context or not. If we did, it should go here.

Reply:
I missed that you were pulling context in List.js to pass to Table. Sorry about that!